runA11yTests.js

```diff
@@ -20,6 +20,34 @@ function resetReportDir( config ) {
 	fs.mkdirSync( config.reportDir, { recursive: true } );
 }
 
+// FIXME: Temporary pa11y workaround for color
+// contrast errors in two groups – first valid, second
+// invalid. Not suitable for production.
+// Switch to axe-core for better accuracy.
+function extractColorContrastInstances( data ) {
+	const colorContrastInstances = [];
+	let addingToArray = false;
+
+	for ( let i = 0; i < data.issues.length; i++ ) {
+		const issue = data.issues[i];
+
+		if ( issue.code === 'color-contrast' ) {
+			if ( !addingToArray ) {
+				// Start adding to the array
+				addingToArray = true;
+			}
+			// Add the current issue to the array
+			colorContrastInstances.push( issue );
+		} else if ( addingToArray && i > 0 && data.issues[i].code !== 'color-contrast' && data.issues[i - 1].code === 'color-contrast' ) {
```

Comment:
I think we could slightly simplify this by removing addingToArray and instead having this:
```
else if ( colorContrastInstances.length > 0 && data.issues[i].code !== 'color-contrast')
```

addingToArray is never used again after the break, so its not really needed. Same with `data.issues[i - 1].code === 'color-contrast'`, also not really needed i believe